python-main.js

```diff
@@ -1356,11 +1358,11 @@ function web_editor(config) {
             // Stop closure of the menu in other local event handlers
             e.originalEvent.keepMenuOpen = true;
         });
-
```

Comment:
The additional line was to separate the main UI button handlers from the other click handlers. 
Also, if we add it again it reduces the diff.